src/mondo_robot_template.py

```diff
@@ -0,0 +1,66 @@
+"""Medgen->Mondo robot template
+
+Create a robot template to be used by Mondo to add MedGen xrefs curated by MedGen.
+
+See also:
+- PR: https://github.com/monarch-initiative/medgen/pull/9
+- Used here: https://github.com/monarch-initiative/mondo/pull/6560
+"""
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Dict, List
+
+import pandas as pd
+
+SRC_DIR = Path(__file__).parent
+PROJECT_DIR = SRC_DIR.parent
+FTP_DIR = PROJECT_DIR / "ftp.ncbi.nlm.nih.gov" / "pub" / "medgen"
+INPUT_FILE = str(FTP_DIR / "MedGenIDMappings.txt")
+OUTPUT_FILE = str(PROJECT_DIR / "ROBOT_addMedGen_fromIngest.tsv")
+
+
+def _prefixed_id_rows_from_common_df(df: pd.DataFrame, mondo_col='mondo_id', xref_col='xref_id') -> List[Dict]:
```

Comment:
I was actually able to use this same exact function for both robot templates: (i) the one generated from Megan's conflicts in https://github.com/monarch-initiative/mondo/pull/6560, and (ii) the ones taken directly from MedGen FTP mappings in this PR.

Reply:
Duplicate comment; marking resolved.